### ISSUE TITLE
Invalid command 'ProxyPassMatch' due to task order of apache module tasks

### ISSUE THREAD

**Issue by HenningTimm** [GlobalDataverseCommunityConsortium/dataverse-ansible#246]
I started to dig myself into deploying dataverse via ansible to a debian machine. First of all, thank you for this nice role! I stumbled upon some problems on the way and I am not sure if these are due to the preparations I did on my debian machine.

I used the develop branch, made a copy of the `defaults/main.yml` that I adapted to my needs and ran everything with default configuration.

During this, I stumbled across the following error in the task `enable apache mods on Debian`:
```
Invalid command 'ProxyPassMatch'

failed: [myhost] (item=proxy) => {"ansible_loop_var": "item", "changed": false, "item": "proxy", "msg": "Error executing /usr/sbin/apache2ctl: AH00526: Syntax error on line 9 of /etc/apache2/sites-enabled/http.proxy.conf:\nInvalid command 'ProxyPassMatch', perhaps misspelled or defined by a module not included in the server configuration\n"}
[...]
```
This seems to be due to the fact that the previous task (`install unified http proxy config`) is already using the ProxyPass command, which seems to prevent installing the modules. Switching the order of these tasks solved the problem for me.

I am not an expert with webserver configuration, so my question is: Is this behavior intentional? Am I doing something wrong here, or should the order of the two tasked be switched? I would be happy to provide a PR for that.

**Comment by donsizemore** [comment]
Hello, I'm afraid the role isn't heavily tested on Debian (typically RockyLinux), so you may well be correct here.

I remembered a commit involving changing the order of placing this template, but it appears to be unrelated:
https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/commit/8af15b787ce105963ed90125f43635b389f697cf

Pull requests are most welcome, and thank you for reporting this problem.

**Comment by HenningTimm** [comment]
We did some research today. Turns out, the `apache2_module` runs a configcheck after each change (cf. https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/blob/develop/tasks/dataverse-apache.yml#L47). When a module is used that is not yet activated, this configcheck fails and this also fails the task (for each loop entry). So using `ProxyPassMatch` would only work after the `proxy` and `http_proxy` modules were already activated.

There are two ways around that:
1. Roll back the changes that swapped the task order.
2. Deactivate the config check. We tried this today: You get one warning after `proxy`, after that the other steps work fine.

I would prefer solution 1, since solution 2 could potentially mask other errors. I would prefer to ask Peter Pallinger about this first (@pallinger hope this is your account, GitHub seems to have some identity issue there). I will create a PR for solution 1.
